docs/source/faqs.md

```diff
@@ -21,3 +21,39 @@ Below series are NOT supported yet:
 - Ascend 910, Ascend 910 Pro B (Ascend-cann-kernels-910) unplanned yet
 
 From a technical view, vllm-ascend support would be possible if the torch-npu is supported. Otherwise, we have to implement it by using custom ops. We are also welcome to join us to improve together.
+
+### 2. How to get our docker containers?
+
+You can get our containers at `Quay.io`, e.g., [<u>vllm-ascend</u>](https://quay.io/repository/ascend/vllm-ascend?tab=tags) and [<u>cann</u>](https://quay.io/repository/ascend/cann?tab=tags).
+
+Plus, if you are in China, you can config your docker proxy to accelerate downloading:
+
+```bash
+vim /etc/docker/daemon.json
+# Add `https://quay.io` to `registry-mirrors` and `insecure-mirrors`
+
+vim /etc/systemd/system/docker.service.d/https-proxy.conf
+# Config proxy
+[Service]
+Environment="HTTP_PROXY=xxx"
+
+sudo systemctl daemon-reload
+sudo systemctl restart docker
+```
```

Comment:
please add the usage for speed proxy `m.daocloud.io/quay.io/ascend/vllm-ascend:v0.7.3rc2`